src/ui/locales/sr.js

```diff
@@ -1,25 +1,25 @@
 // @flow
 
 const locale = {
-    "AttributionControl.ToggleAttribution": "Укључите приписивање",
+    "AttributionControl.ToggleAttribution": "Укључи приписивање",
     "AttributionControl.MapFeedback": "Повратне информације о мапи",
-    "FullscreenControl.Enter": "Унесите цео екран",
-    "FullscreenControl.Exit": "Изађите преко целог екрана",
-    "GeolocateControl.FindMyLocation": "Пронађите моју локацију",
+    "FullscreenControl.Enter": "Прикажи на целом екрану",
+    "FullscreenControl.Exit": "Прекини приказ на целом екрану",
+    "GeolocateControl.FindMyLocation": "Пронађи моју локацију",
     "GeolocateControl.LocationNotAvailable": "Локација није доступна",
-    "LogoControl.Title": "Мапбок лого",
+    "LogoControl.Title": "© Mapbox лого",
```

Comment:
Transliteration of company names is rare.

Reply:
We don't have the copyright symbol in other languages